buffer.go

```diff
@@ -380,11 +380,7 @@ func levelledPoolIndex(sz int) int {
 }
 
 var (
-	levelsBufferPool           bufferPool
-	compressedPageBufferPool   bufferPool
-	uncompressedPageBufferPool bufferPool
-	pageOffsetsBufferPool      bufferPool
-	pageValuesBufferPool       [8]bufferPool
+	bytesPool bufferPool
```

Comment:
At first I felt like this name communicated something like "a pool of one-byte values". 

Have you considered an alternative name? Maybe `buffers`?

Reply:
works for me. changed!